operator/cmd/sidecar/sidecar.go

```diff
@@ -106,6 +108,7 @@ func Command() *cobra.Command {
 
 	// users flags
 	cmd.Flags().BoolVar(&watchUsers, "watch-users", false, "Specifies if the sidecar should watch and configure superusers based on a mounted users file.")
+	cmd.Flags().BoolVar(&skipClusterConfigurationSync, "skip-cluster-configuration-sync", false, "Specifies if the sidecar should sync superuser cluster configuration list after users passwords are synced.")
```

Comment:
This makes it seem like the config watch perform cluster config reconciliation. What about:

```suggestion
	cmd.Flags().BoolVar(&skipClusterConfigurationSync, "no-set-superusers", false, "Specifies if the sidecar should sync the superuser cluster configuration with watched users or not")
```